src/inc/msquic.h

```diff
@@ -109,6 +109,7 @@ typedef enum QUIC_CREDENTIAL_FLAGS {
     QUIC_CREDENTIAL_FLAG_IGNORE_REVOCATION_OFFLINE              = 0x00001000, // Schannel only currently
     QUIC_CREDENTIAL_FLAG_SET_ALLOWED_CIPHER_SUITES              = 0x00002000,
     QUIC_CREDENTIAL_FLAG_USE_PORTABLE_CERTIFICATES              = 0x00004000, // OpenSSL only currently
```

Comment:
Is this comment still correct? I thought @ThadHouse did work to enable portable certs on schannel

Reply:
I think you might be right, but I don't want to contaminate this PR with fixing.

Reply:
Yeah this comment is not correct anymore and could be removed.

Reply:
I went ahead and made the change.